new-content/events/summer-school-2021.yaml

```diff
@@ -28,44 +28,44 @@ mosaic:
   title: 'Quantum Learning & More'
   tiles:
     - position: 'first'
+      title: 'Qiskit Textbook'
+      description: 'The Qiskit Textbook is a free digital open source textbook that will teach you the concepts of quantum computing while you learn to use Qiskit.'
+      image: '/images/events/seminar-series/mosaic-experts.png'
+      cta:
+        url: 'https://qiskit.org/textbook/preface.html'
+        label: 'Read the textbook'
+        segment:
+          action: events-summer-school > mosaic > read-the-textbook
+
+    - position: 'second'
       title: 'IBM Quantum'
       description: 'IBM offers cloud access to the most advanced quantum computers available. Learn, develop, and run programs with our quantum applications and systems.'
-      image: '/images/events/seminar-series/mosaic-experts.png'
+      image: '/images/events/summer-school/IBM_System_One_display.jpg'
```

Comment:
Please, follow the same kebab-case style we are using for all the image names: `IBM_System_One_display` -> `ibm-system-one-display`